Gradata/docs/research/graduation-quality-2026-05-21.md

```
@@ -0,0 +1,231 @@
+|--------|-------|-----------------|
+| Total lessons | 8 | Tiny corpus (test/dev environment) |
+| Promoted (≥PATTERN) | 2 | 25% graduation rate |
+| Sessions in DB | 1 | Single session only — no statistical power |
+| Lesson applications | 162 | High-frequency applicability test data |
+| Meta-rules | 0 | No meta-level clustering detected |
+| Total RULE_GRADUATED events | 84 | High graduation churn |
+| Organic PATTERN→RULE | **0** | **CRITICAL: Zero organic rule tier advancement** |
+
+### 2. Graduation Event Histogram
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)